src/uct/ib/dc/base/dc_iface.h

```diff
@@ -45,6 +45,14 @@ typedef struct uct_dc_dci {
 } uct_dc_dci_t;
 
 
+typedef struct uct_dc_fc_request {
+    uct_rc_fc_request_t           super;
+    struct ibv_ah                 *ibv_ah;
```

Comment:
looks like ibv_ah field should go into a new struct:  uct_dc_**verbs**_fc_request_t